s3auth-hosts/src/test/java/com/s3auth/hosts/DefaultResourceTest.java

```diff
@@ -56,6 +58,19 @@
  * @checkstyle ClassDataAbstractionCoupling (500 lines)
  */
 public final class DefaultResourceTest {
+    /**
+     * Test bucket.
+     */
+    public static final String BUCKET = "abcdef";
```

Comment:
it's not an API, and it's used only twice - no need to extract the value as a fileld